index.php

```diff
@@ -37,6 +34,7 @@
 });
 
 function updateDropdown(District, City, Street) {
+  let html_code;

```

Comment:
I created a html_code variable in the local scope. I've noticed you're mixing up local and global scope on the html_code variable. Better keep that local because all html templates are function specific anyway.

Also it is best practice to keep everything scoped local until unless it is absolutely necessary to keep things in global scope to avoid debugging headaches later.

also you can see I used let instead of var. They both works same in this code but there are nuances between what a var, let and const means. You can learn about them later as you write more JS, for now just think of let as a cleaner/recommended version of var.